web/app/js/components/TopEventTable.jsx

```diff
@@ -65,12 +65,20 @@ const topColumns = (resourceType, ResourceLink) => [
     width: "128px",
     sorter: (a, b) => numericSort(a.successRate.get(), b.successRate.get()),
     render: d => successRateWithMiniChart(d.get())
+  },
+  {
+    title: "Tap",
+    key: "tap",
+    className: "numeric",
+    width: topMetricColWidth,
```

Comment:
how do you feel about 30px here? The column spacing feels a little wide.

![screen shot 2018-09-14 at 10 29 11 am](https://user-images.githubusercontent.com/549258/45565682-19b04900-b809-11e8-9b14-77dc353d1a1f.png)